tests/benchmarks/power_consumption/adc/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  tags: ppk_power_measure
+
+tests:
+  test.power_consumption.adc_nrf54l:
+    platform_allow:
+      - nrf54l15dk/nrf54l15/cpuapp
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+       - "../test_measure_power_consumption.py::test_measure_and_data_dump_placeholder"
```

Comment:
Please change path to: ${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py:...

Reply:
changed